### DISCUSSION TITLE
No error message when composite key constraint is violated

### DISCUSSION THREAD

**Discussion by sarthakgaur** [directus/directus#9249] [Archive]
directus: v9.0.0-rc.100 

I created a constraint on my table manually by using a command similar to this:

`ALTER TABLE table_name ADD CONSTRAINT unique_col1_col2 UNIQUE (col1, col2);`

When I try to create a new item from the directus UI that violates this constraint, I don't see any error messages either in the UI or the server logs. When I click the "Save" button on the top right corner, nothing happens.

**Comment by azrikahar** [comment]
This should be what was reported in #8028 👍
